### PR TITLE
modify the procedures document to say that we propose meetings with open issues

### PR DESCRIPTION
**Pull request by nikomatsakis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/meeting-proposal.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ assignees: ''
 # Directions
 
 If you'd like to propose a topic for a future meeting, you've come to
-the right place! To do so, please write-up your proposal in a gist,
-hackmd, or other such format. Then open this issue and replace the
-parentheiszed text below as appropriate. We'll discuss your proposal
-in the next planning meeting! (And please do feel free to attend!)
-Thanks <3.
+the right place! To do so, please write-up your proposal in a hackmd,
+gist, patsebin etc following [this template]. Then open this issue and
```
Comment (Centril) [inline .github/ISSUE_TEMPLATE/meeting-proposal.md]:
```suggestion
gist, pastebin etc following [this template]. Then open this issue and
```

Comment (davidtwco) [inline .github/ISSUE_TEMPLATE/meeting-proposal.md]:
```suggestion
gist, pastebin, etc following [this template]. Then open this issue and
```

File: .github/ISSUE_TEMPLATE/meeting-proposal.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ assignees: ''
 # Directions
 
 If you'd like to propose a topic for a future meeting, you've come to
-the right place! To do so, please write-up your proposal in a gist,
-hackmd, or other such format. Then open this issue and replace the
-parentheiszed text below as appropriate. We'll discuss your proposal
-in the next planning meeting! (And please do feel free to attend!)
-Thanks <3.
+the right place! To do so, please write-up your proposal in a hackmd,
```
Comment (Centril) [inline .github/ISSUE_TEMPLATE/meeting-proposal.md]: I would recommend linking to each of hackmd, gist, and pastebin.

File: .github/ISSUE_TEMPLATE/meeting-proposal.md (hunk 1 above)
Comment (Centril) [reply .github/ISSUE_TEMPLATE/meeting-proposal.md]: Damn! 32 seconds late :D

**Pull request merged**